package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "truex-shared",
-  "version": "1.7.10",
+  "version": "1.7.11",
```

Comment:
I am not seeing PRs for TAR-HTML5 and Firetv-Webview to update their dependencies as well.

Reply:
yea, I guess I should still update the version even if it is not using anything from here-- just to make the story complete.

(Just for the record, this is an optional event, and existing publishers do not have to update TAR/integration to use the new feature)